code/web/src/modules/user/Profile.js

```diff
@@ -31,16 +35,42 @@ const Profile = (props) => (
     </Grid>
 
     <Grid>
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+        {/*<Tile style={{}} image={icon}></Tile>*/}
+        <ImageTile width={300} height={530} shadow={level1} image={`${ APP_URL }/images/stock/men/1.jpg`} />
+      </GridCell>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
-
+        <p style={{ marginBottom: '0.5em' }}>Personal description goes here</p>
+        {/*Description text should be dynamic from user details ie props.user.details.description */}
+      </GridCell>
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
-
-        <Link to={userRoutes.subscriptions.path}>
-          <Button theme="primary">Subscriptions</Button>
-        </Link>
-
+        <p style={{ color: grey2, marginBottom: '2em' }}>123 West 5th St City, State 80654</p>
+        {/*Address text should be dynamic from user details ie props.user.details.shipping */}
+      </GridCell>
+    </Grid>
+    <Grid>
+      {
+        this.props.orderHistory.isLoading
+          ? <Loading/>
+          : this.props.orderHistory.list.length > 0
+              ? this.props.orderHistory.list.map(product => (
+                  <GridCell key={product.id} style={{ margin: '2em', float: 'left' }}>
+                    <H3>{product.name}</H3>
+                    <p>{product.description}</p>
+                    {/* Items will also have a boolean value to indicate if item was kept */}
```

Comment:
We need to be sure to communicate the addition of the Boolean value to @EHale64, @TravisGM92, and @fieldstyler so that they can make changes to the structure of how `product` holds that data.